### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by Sammers21**

### CODE REVIEW

File: src/main/java/io/vertx/mqtt/package-info.java [hunk 1]
```diff
@@ -1,7 +1,47 @@
 /**
  * = Vert.x MQTT client
  *
- * include::{@link io.vertx.mqtt}[]
+ * == Using Vert.x MQTT client
+ *
+ * TODO
```
Comment (Sammers21) [inline src/main/java/io/vertx/mqtt/package-info.java]: If not sure how anyone can use vertx-mqtt-artifact. I think the chapter should be filled when there will be an opportunity to get this artifact from any repository.

Comment (Sammers21) [inline src/main/java/io/vertx/mqtt/package-info.java]: For example in mqtt-server we have:
image

Comment (Sammers21) [inline src/main/java/io/vertx/mqtt/package-info.java]: I could do something like this.

Comment (ppatierno) [inline src/main/java/io/vertx/mqtt/package-info.java]: Leave just blank for now.

Comment (Sammers21) [inline src/main/java/io/vertx/mqtt/package-info.java]: You mean to delete even "TODO"? And leave this chapter like:
```
== Using Vert.x MQTT 
==Getting started
.....
```
?

Comment (ppatierno) [inline src/main/java/io/vertx/mqtt/package-info.java]: yep for now ;)

Comment (Sammers21) [inline src/main/java/io/vertx/mqtt/package-info.java]: ok

**Comment by Sammers21** [comment]
Suggestions accepted and was done

**Comment by ppatierno** [comment]
Can you squash into just the one commit "add initial documentation" ?

**Comment by Sammers21** [comment]
Sure, I have just done it.

**Pull request merged**